src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.CommonSVTControls.ps1

```diff
@@ -230,7 +336,10 @@ class CommonSVTControls: ADOSVTBase {
                         if ($groupFoundWithExcessivePermissions -eq $true) {
                             $excessivePermissionsGroupObj = @{}
                             $excessivePermissionsGroupObj['Group'] = $broaderGroup.displayName
-                            $excessivePermissionsGroupObj['ExcessivePermissions'] = $($excessivePermissionsPerGroup.displayName -join ', ')                            
+                            $excessivePermissionsGroupObj['ExcessivePermissions'] = $($excessivePermissionsPerGroup.displayName -join '; ') 
+                            $excessivePermissionsGroupObj['Descriptor'] = $broaderGroupResponseObj[0].dataProviders.'ms.vss-admin-web.security-view-permissions-data-provider'.identityDescriptor
+                            $excessivePermissionsGroupObj['PermissionSetToken'] = $excessivePermissionsPerGroup[0].token
+                            $excessivePermissionsGroupObj['PermissionSetId'] = $excessivePermissionsPerGroup[0].namespaceId                           
```

Comment:
We dont need to store permissionSetId, we have standard namespace in global variable $repoPermissionSetId